source/common/config/delta_subscription_state.cc

```diff
@@ -1,5 +1,7 @@
 #include "common/config/delta_subscription_state.h"
 
+#include "common/common/assert.h"
```

Comment:
Generally "any includes present in the related header do not need to be included again in the related cc (i.e., foo.cc can rely on foo.h's includes)." but it doesn't do harm so just consider it in future?

Reply:
Ah, did not know that. Thanks I'll keep that in mind.